catalyst/utils/meters/averagevaluemeter.py

```diff
@@ -41,17 +42,22 @@ def add(self, value) -> None:
         """
         self.val = value
         self.n += 1
+        self.n_samples += batch_size
 
         if self.n == 1:
             self.mean = 0.0 + value  # Force a copy in torch/numpy
             self.std = np.inf
             self.mean_old = self.mean
             self.m_s = 0.0
         else:
-            self.mean = self.mean_old + (value - self.mean_old) / float(self.n)
-            self.m_s += (value - self.mean_old) * (value - self.mean)
+            self.mean = self.mean_old + (
+                value - self.mean_old
+            ) * batch_size / float(self.n_samples)
+            self.m_s += (
+                (value - self.mean_old) * (value - self.mean) * batch_size
+            )
             self.mean_old = self.mean
-            self.std = np.sqrt(self.m_s / (self.n - 1.0))
+            self.std = np.sqrt(self.m_s / (self.n_samples - 1.0))
```

Comment:
could we write tests to check if `mean`, `std` computation is correct?

Reply:
Done